docs/src/development/local/_index.md

```diff
@@ -126,17 +127,30 @@ If you want your local development environment to be enclosed
 so your main system remains unaffected by the build and its dependencies, 
 you can use a local virtual machine.
 
-To build your site locally, run this command:
+To build your site locally:
 
-```bash
-platform build
-```
+1. Run the following command:
+
+   ```bash
+   platform build
+   ```
+
+2. When prompted, specify the source directory and the build destination.
 
-When prompted, specify the source directory and the build destination.
-Your app is then built locally.
+  Your app is built locally.
+  At the root of your local project directory, a `_www` subdirectory is created.
+  It is a symbolic link pointing to the currently active build located in the `.platform/local/builds` directory.
 
-At the root of your local project directory, a `_www` subdirectory is created.
-It is a symbolic link pointing to the currently active build located in the `.platform/local/builds` directory.
+3. To check that the build was successful:
```

Comment:
:heart_eyes: Very well summarized! It gives a nice introduction to the sub-topics.